modules/openapi-generator/src/main/resources/python-pydantic-v1/signing.mustache

```diff
@@ -76,6 +76,7 @@ class HttpSigningConfiguration:
         a private key. The private key is used to sign HTTP requests.
     :param private_key_passphrase: A string value specifying the passphrase to decrypt
         the private key.
+    :param private_key_string: Specify the API key as string.
```

Comment:
what about `modules/openapi-generator/src/main/resources/python/signing.mustache`? does that need to be updated as well?

Reply:
python does not require it, because get_http_signature_headers method is accessible wherever we refer the class HttpSigningConfiguration.

Reply:
Sorry python related changes was committed accidentally. I have removed it.